TrustModel_Personas.md

```diff
@@ -0,0 +1,179 @@
+
+Example : Cloud Service Provider (CSP) Infrastructure as a Service (IaaS) Site Reliability Engineer
+(SRE)
+
+### Orchestration Operator
+This persona has privileges within the Orchestration/Cluster.
+They are responsible for deploying a solution into a particular cloud native environment and 
+managing the orchestration environment.
+For managed cluster this would also include the administration of the cluster control plane.
+- control availability of service
+- control webhooks and deployment of workloads
+- control availability to workload of cluster resources (data/networking/storage) and cluster 
+  services (Logging/Monitoring/Load Balancing)
+- control the deployment of boot images initially required by the TEE during initialisation. 
+ These boot images once initialised will receive the workload.
+
+Example : A kubernetes administrator responsible for deploying pods to a cluster and 
+maintaining the cluster.
+
+### Workload Provider
+This persona develops, builds and defines the solution, pods and containers intended to provide 
+the Application or Workload which in turn provides value to the data owners (customers and clients).
+The workload provider and data owner could be part of same company/organisation but 
+following the least privilege principle the application provider should not be able to view or 
+manipulate end user data without informed consent.
+- Owner of business logic containers. These may contain proprietary algorithms, models or secrets
+- Need to prove to customer aspects of compliance.
+- Defines what the solution requires in order to run and maintain compliance (resources, utility 
+  containers/services,  storage).
+- Defines the method of verifying the container images to be used and provides artifacts to 
+  allow verification to be completed within the TEE.
+- provide the boot images initially required by the TEE during 
+  initialisation or designates a trusted party to do so.
+- provide the attestation verification service, or designate a trusted party to provide the 
+  attestation verification service.
+
+Example : 3rd party software vendor, cloud solution provider
+
+### Container Image Provider
+
+This persona is responsible for the part of the supply chain that builds container images and 
+provides them for use by the solution. Since a workload can 
+be composed of multiple containers, there may be multiple container image providers, some 
+will be closely connected to the workload provider (business logic containers), others more 
+independent to the workload provider (side car containers). The container image provider is expected
+to use a mechanism to allow provenance of container image to be established when a 
+workload pulls in these images at deployment time. This can take the form of signing or encrypting 
+the container images.
+- Builds container images
+- Signs or encrypts the images
+- Provides the signature or public encryption keys securely to the workload provider by a trusted 
```

Comment:
Update key terminology. (Suggested parenthetic on the second bullet might be too specific but those components are defined by our system.)
 
```suggestion
- Publishes the signature verification key (public key).
- Provides any decryption keys through a secure channel (generally to a key management system controlled by a Key Broker Service).
```

Reply:
Yes I like splitting this out , I added a third point to cover any more generic things beyond keys and the images